### ISSUE TITLE
Question about SSDFocalLoss?

### ISSUE THREAD

**Issue by feixiangdekaka** [donnyyou/torchcv#64]
in the code :

        logit = F.softmax(x,dim=-1)
        logit = logit.clamp(1e-7, 1.-1e-7)
        conf_loss_tmp = -1 * t.float() * torch.log(logit)
        conf_loss_tmp = alpha * conf_loss_tmp * (1-logit)**gamma
        conf_loss = conf_loss_tmp.sum()

the positive  and negative  has the same alpha=0.25,some implement is when positive alpha =0.25,negative=1-0.25,what is the difference?
